db/migrate/20170924065938_create_restaurant_reviews_metadata.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRestaurantReviewsMetadata < ActiveRecord::Migration[5.1]
+  def change
+    create_table :restaurant_reviews_metadata do |t|
```

Comment:
Why not on the Review model?

Reply:
wontfix - agreed f2f

Reply:
Curious, what's the reason for this?

Reply:
this is great. i really like it. 
while it may seem excessive for a pet project, in a larger system this really saves you from models becoming a huge pile of garbage and from memory intensive and slow queries a la `SELECT * FROM restaurants`.
💯